openslides_backend/action/actions/meeting/import_.py

```diff
@@ -324,7 +324,9 @@ def update_meeting_and_users(self, instance: Dict[str, Any]) -> None:
         # generate passwords
         for entry in json_data["user"].values():
             if entry["id"] not in self.merge_user_map:
-                entry["password"] = self.auth.hash(get_random_string(10))
+                if not entry.get("default_password"):
```

Comment:
After talking with @emanuelschuetze : Create always a new default_password and set it, i.e. never import the default_password.
We don't need to send the password automatically to the users, the admin will and can select users to send invitation mails.